app/controllers/restaurants_controller.rb

```diff
@@ -1,13 +1,48 @@
 class RestaurantsController < ApplicationController
-  def index
-  end
+  	
+  	def index
+  		@restaurants = Restaurant.all
+  	end
+
+  	def show
+  		@restaurant = Restaurant.find(params[:id])
+  	end
+
+  	def new
+  		@restaurant = Restaurant.new
+  	end
+
+  	def create
+  		Restaurant.create(params.require(:restaurant)
+  			.permit(:name, :website, :created_at, :updated_at, 
+  			:hours, :low_price, :high_price))
+  		@restaurant = Restaurant.all
```

Comment:
Will look up these lines on the train.  Heidi

 

From: Brian Gates [mailto:notifications@github.com] 
Sent: Tuesday, June 16, 2015 1:52 AM
To: TheIronYard--Orlando/ror_granular_app
Cc: Heidi L. Roude
Subject: Re: [ror_granular_app] ahhhhh, the domino errors and subsequent headaches (#14)

 

In app/controllers/restaurants_controller.rb <https://github.com/TheIronYard--Orlando/ror_granular_app/pull/14#discussion_r32492848> :

> +            @restaurants = Restaurant.all
> +     end
> +
> +     def show
> +            @restaurant = Restaurant.find(params[:id])
> +     end
> +
> +     def new
> +            @restaurant = Restaurant.new
> +     end
> +
> +     def create
> +            Restaurant.create(params.require(:restaurant)
> +                    .permit(:name, :website, :created_at, :updated_at, 
> +                    :hours, :low_price, :high_price))
> +            @restaurant = Restaurant.all

what does line 19 do?

—
Reply to this email directly or view it on GitHub <https://github.com/TheIronYard--Orlando/ror_granular_app/pull/14/files#r32492848> .Image removed by sender.